tests/index.js

```diff
@@ -9,12 +9,30 @@
 /**
  * tests/index.js
  */
+/* eslint camelcase: 0 */
 const expect = require('chai').expect;
+const createTTE = require('../index.js').createTTE;
```

Comment:
you can use destructuring here
const { createTTE } = require('../index.js');
just a matter of preference though